# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/VRProcessParent.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/vr/ipc/VRProcessParent.cpp
--- a/gfx/vr/ipc/VRProcessParent.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/vr/ipc/VRProcessParent.cpp	Sun May 26 14:30:14 2019 +0000
@@ -11,6 +11,7 @@
 #include "mozilla/gfx/GPUChild.h"
 #include "mozilla/ipc/ProtocolTypes.h"
 #include "mozilla/ipc/ProtocolUtils.h"  // for IToplevelProtocol
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/TimeStamp.h"          // for TimeStamp
 #include "mozilla/Unused.h"
 #include "ProcessUtils.h"
@@ -81,7 +82,7 @@
     return !!mVRChild;
   }
 
-  int32_t timeoutMs = gfxPrefs::VRProcessTimeoutMs();
+  int32_t timeoutMs = StaticPrefs::VRProcessTimeoutMs();
 
   // If one of the following environment variables are set we can effectively
   // ignore the timeout - as we can guarantee the compositor process will be